File: single-language-analysis/dataset-files/python/test_smell_015.py
import pytest

class TestGameTest:
    def test_smell_015(self):
        game = Game(n3win_algorithm, CALCULATION_DEPTH)
        field = [[None for _ in range(5)] for _ in range(5)]

        game.bot_place(field, Chip.RED)
        counter = 0
        for x in range(SIZE):
            for y in range(SIZE):
                if field[x][y] == Chip.RED:
                    counter += 1
        assert counter == 1

File: single-language-analysis/dataset-files/java/TestSmell015.java
// Original URL: https://github.com/muenchhausen/3wins/blob/57b86a71a57b9a62f811d7d568b11d9d676821fd/GameTest.java#L67-L86

public class TestSmell015 {
    @Test
    public void bot_should_place_a_new_chip ()
    {
        Game game = new Game(n3winAlgorithm,CALCULATION_DEPTH);
        Chip field[][] = new Chip[5][5];

        game.botPlace(field, Chip.RED);
        int counter=0;
        for (int x=0; x<SIZE; x++) {
            for (int y=0; y<SIZE; y++) {
                if(field[x][y]==Chip.RED){
                    counter++;
                }
            }
        }
        assertEquals(1,counter);

        
       
    }
}
